index.d.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ *
+ * Returns the address ipv6 of the internet-facing interface, as determined from the default gateway. When the address cannot be determined for any reason, null will be returned.
+ * @returns string of the ip address
+ *
+ * @example
+ * console.log(await internalIp.v6());
+ * //=> 'fe80::1'
+ *
+ */
+export function v6(): Promise<string>;
```

Comment:
These doesn't really make much sense as named exports. I think it would be better to just export `internalIp` as a default export with properties.

Reply:
in the package there is no default export

https://github.com/sindresorhus/internal-ip/blob/master/index.js#L43-L47

Reply:
There are no real "named exports" in CommonJS. `module.exports` is a default export. `module.exports.foo` is just a property on the `module.export` object.